frontend/src/app/patients/Components/ManageEmails.tsx

```diff
@@ -136,15 +144,20 @@ const ManageEmails: React.FC<Props> = ({
                 onEmailChange(idx, e.target.value)
               }
             />
-            <div className="flex-align-self-end">
-              <button
-                className="usa-button--unstyled padding-105 height-5"
-                onClick={() => onEmailRemove(idx)}
-                aria-label={`Delete email ${email}`.trim()}
-              >
-                <FontAwesomeIcon icon={"trash"} className={"text-error"} />
-              </button>
-            </div>
+            {idx > 0 ? (
+              <div className="flex-align-self-end">
+                <button
+                  className="usa-button--unstyled padding-105 height-5"
+                  data-testid={`delete-email-${idx}`}
+                  onClick={() => onEmailRemove(idx)}
+                  aria-label={`Delete email ${email}`.trim()}
```

Comment:
If we wanted to trim the whitespace from the user inputted email value I think we have to do `Delete email ${email.trim()}`